multicluster/test/e2e/antreapolicy_test.go

```diff
@@ -0,0 +1,167 @@
+			allPodsPerCluster = append(allPodsPerCluster, antreae2e.NewPod(ns, podName))
+			podsByNamespace[ns] = append(podsByNamespace[ns], antreae2e.NewPod(ns, podName))
+		}
+	}
+	for clusterName := range data.clusterTestDataMap {
+		d := data.clusterTestDataMap[clusterName]
+		k8sUtils, err := antreae2e.NewKubernetesUtils(&d)
+		failOnError(err, t)
+		_, err = k8sUtils.Bootstrap(perClusterNamespaces, perNamespacePods)
+		failOnError(err, t)
+		clusterK8sUtilsMap[clusterName] = k8sUtils
+	}
+}
+
+// tearDownForPolicyTest deletes the test Namespaces specific for policy tests.
+func tearDownForPolicyTest() {
+	for _, k8sUtils := range clusterK8sUtilsMap {
+		k8sUtils.Cleanup(perClusterNamespaces)
+	}
+}
+
+func testMCAntreaPolicy(t *testing.T, data *MCTestData) {
+	data.testAntreaPolicyCopySpanNSIsolation(t)
+}
+
+// testAntreaPolicyCopySpanNSIsolation tests that after applying a ResourceExport of an ACNP
+// for Namespace isolation, strict Namespace isolation is enforced in each of the member clusters.
+func (data *MCTestData) testAntreaPolicyCopySpanNSIsolation(t *testing.T) {
+	setup := func() {
+		err := data.deployACNPResourceExport(acnpIsolationResourceExport)
+		failOnError(err, t)
+	}
+	teardown := func() {
+		err := data.deleteACNPResourceExport(acnpIsolationResourceExport)
+		failOnError(err, t)
+	}
+	reachability := antreae2e.NewReachability(allPodsPerCluster, antreae2e.Dropped)
+	reachability.ExpectAllSelfNamespace(antreae2e.Connected)
+	testStep := &antreae2e.TestStep{
+		Name:         "Port 80",
+		Reachability: reachability,
+		Ports:        []int32{80},
+		Protocol:     v1.ProtocolTCP,
+	}
+	testCaseList := []*antreae2e.TestCase{
+		{
+			Name:  "ACNP strict Namespace isolation for all clusters",
+			Steps: []*antreae2e.TestStep{testStep},
+		},
+	}
+	executeTestsOnAllMemberClusters(t, testCaseList, setup, teardown)
+}
+
+func executeTestsOnAllMemberClusters(t *testing.T, testList []*antreae2e.TestCase, setup, teardown func()) {
+	setup()
+	time.Sleep(networkPolicyDelay)
+	for _, testCase := range testList {
+		log.Infof("Running test case %s", testCase.Name)
+		for _, step := range testCase.Steps {
+			log.Infof("Running step %s of test case %s", step.Name, testCase.Name)
+			reachability := step.Reachability
+			if reachability != nil {
+				for clusterName, k8sUtils := range clusterK8sUtilsMap {
+					if clusterName == leaderCluster {
+						// skip traffic test for the leader cluster
+						continue
+					}
+					if _, err := k8sUtils.GetACNP(acnpName); err != nil {
+						t.Errorf("Failed to get ACNP to be replicated in cluster %s", clusterName)
+					}
+					start := time.Now()
+					k8sUtils.Validate(allPodsPerCluster, reachability, step.Ports, step.Protocol)
+					step.Duration = time.Now().Sub(start)
+					_, wrong, _ := step.Reachability.Summary()
+					if wrong != 0 {
+						t.Errorf("Failure in cluster %s -- %d wrong results", clusterName, wrong)
+						reachability.PrintSummary(true, true, true)
```

Comment:
not introduced in this PR, but I see all callers of reachability.PrintSummary() are using `(true, true, true)`, maybe we can remove all three parameters. do you think if this can be enhanced by https://github.com/antrea-io/antrea/pull/3286 by enhao?

Reply:
Sure. /cc @enhaocui for awareness.